scripts/stack/configure-connectors.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Should we move this script into `docker.elastic.co/enterprise-search/elastic-connectors` image and thus run this initialization inside the container, as an entrypoint? Docker would be the only pre-requisite on the local machine, in this case.

I read it seems we need an entrypoint (see https://github.com/elastic/enterprise-search-team/issues/6469), so we may think about adding a "smart" entrypoint, which could also generate connectors configuration before starting them (in case the container is created by a human user).

Thoughts?